### PR TITLE
[New sample] Show labels on layers in 3D

### PR DESCRIPTION
**Pull request by williambohrmann3**
# Description

New sample for showing labels on layers in 3D scenes.

## Type of change

- [ ] Bug fix
- [x] New sample implementation
- [ ] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [x] WPF .NET 6
- [x] WPF Framework
- [x] WinUI
- [x] MAUI Windows
- [x] MAUI Android
- [ ] MAUI iOS
- [x] MAUI Mac Catalyst

## Checklist

- [x] Self-review of changes
- [x] Runs and compiles on all active platforms
- [x] Branch is up to date with the latest main/v.next
- [x] All merge conflicts have been resolved
- [x] There are no warnings related to changes
- [x] No unrelated changes have been made to any other code or project files
- [x] `sample_sync.py` runs without making changes
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] Code is commented with correct formatting
- [x] All variable and method names are good and make sense
- [x] There is no leftover commented code
- [x] Screenshots are correct size and display in description tab (800 x 600 on Windows, 600 height mobile screenshots for MAUI)

### CODE REVIEW

File: src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            try
+            {
+                // Try to load the default portal, which will be ArcGIS Online.
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+
+                // Create the portal item.
+                PortalItem websceneItem = await PortalItem.CreateAsync(portal, ItemId);
+
+                // Create the scene.
+                MySceneView.Scene = new Scene(websceneItem);
```
Comment (mstefarov) [inline src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs:58]: IMO this sample could be simplified to just creating a Scene from an item URL, e.g.

```cs
        private const string ItemUrl = "https://www.arcgis.com/home/item.html?id=850dfee7d30f4d9da0ebca34a533c169";
// ...
        MySceneView.Scene = new Scene(new Uri(ItemUrl));
```

File: src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/readme.metadata.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "web scene"
+  ],
+  "redirect_from": [
+    "/net/latest/maui/sample-code/ShowLabelsOnLayer3D.htm"
+  ],
+  "relevant_apis": [
+    "PortalItem",
+    "Scene",
+    "SceneView"
+  ],
```
Comment (mstefarov) [inline src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/readme.metadata.json]: I'd consider `LabelDefinition` and `ArcadeLabelExpression` to be very relevant here, much more so than `PortalItem`.  Adding items here will help users discover samples by adding links to the API reference.

File: src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+using Esri.ArcGISRuntime.Portal;
+
+namespace ArcGIS.Samples.ShowLabelsOnLayer3D
+{
+    [ArcGIS.Samples.Shared.Attributes.Sample(
+        "Show labels on layer in 3D",
+        "Scene",
+        "add later",
+        "")]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData()]
```
Comment (mstefarov) [inline src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs:35]: Doesn't look like this sample actually uses any offline data -- you can remove this attribute.

File: src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/readme.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## How to use the sample
+
+Pan and zoom to explore the scene. Notice the labels showing installation dates of features in the 3D gas network.
+
+## How it works
+
+To open a web scene from a portal item:
+
+1. Create an `ArcGISScene` from a portal, then load.
```
Comment (ZackAllen) [inline src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/readme.md:19]:
```suggestion
1. Create an `Scene` from a portal, then load.
```
It is just called `Scene` in the .NET API.

File: src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                // Try to load the default portal, which will be ArcGIS Online.
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+
+                // Create the portal item.
```
Comment (ZackAllen) [inline src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs:56]: Comment for every line is too much. We should cut back on the comments.

File: src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks;
+using Esri.ArcGISRuntime.Tasks.Offline;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.ArcGISServices;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
```
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs:19]: Codemaid has not been run on this file. Re-reun codemaid and xaml styler on everything with the correct configurations.

File: src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/readme.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+* Scene
+* SceneView
+
+## About the data
+
+This sample shows a [New York City infrastructure](https://www.arcgis.com/home/item.html?id=850dfee7d30f4d9da0ebca34a533c169) scene hosted on ArcGIS Online.
+
+## Tags
+
+portal, scene, web scene
```
Comment (ZackAllen) [inline src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/readme.md:38]: We should add tags related to labelling and 3d.

File: src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+﻿<UserControl x:Class="ArcGIS.WPF.Samples.ShowLabelsOnLayer3D.ShowLabelsOnLayer3D"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
```
Comment (ZackAllen) [inline src/WPF/WPF.Viewer/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml:5]: Grid is unneeded.

**Review by ZackAllen — CHANGES_REQUESTED** [review]

**Review by ZackAllen — CHANGES_REQUESTED** [review]

**Review by mstefarov — APPROVED** [review]

File: src/WPF/WPF.Viewer/App.xaml [hunk 9]
```diff
@@ -1,8 +1,9 @@
-﻿<Application x:Class="ArcGIS.WPF.Viewer.App"
```
Comment (ZackAllen) [inline src/WPF/WPF.Viewer/App.xaml:1]: unrelated change

File: src/WPF/WPF.Viewer/Resources/ControlStyles.xaml [hunk 10]
```diff
@@ -1,6 +1,7 @@
-﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```
Comment (ZackAllen) [inline src/WPF/WPF.Viewer/Resources/ControlStyles.xaml:1]: unrelated change

**Review by ZackAllen — CHANGES_REQUESTED** [review]

**Review by ZackAllen — APPROVED** [review]

**Pull request merged**